nav2_core/include/nav2_core/behavior_tree_navigator.hpp

```diff
@@ -128,7 +128,8 @@ class NavigatorBase
     const std::vector<std::string> & plugin_lib_names,
     const FeedbackUtils & feedback_utils,
     nav2_core::NavigatorMuxer * plugin_muxer,
-    std::shared_ptr<nav2_util::OdomSmoother> odom_smoother) = 0;
+    std::shared_ptr<nav2_util::OdomSmoother> odom_smoother,
+    std::shared_ptr<BT::StdCoutLogger> logger_cout_) = 0;
```

Comment:
Why not the abstract logger so folks could pass in any logger they want (or have option in the higher level node to select the logger type they prefer)?